android/new-project.md

```diff
@@ -16,7 +16,7 @@
 
 - [ ] Enable **Automatically delete head branches** so there is no need to delete branches manually after each PR merge.
 
-- [ ] Set branch protection rules to `develop` and `main` branches exactly as shown in the following screenshot:
+- [ ] Set branch protection rules to `develop` and `master` branches exactly as shown in the following screenshot:
```

Comment:
znamena to ze budeme pouzivat `master` branch misto `main` ?

Reply:
Git vytvoří automaticky master, tak jsem to v readme přepsal, aby to korespondovalo. Mám to nechat jako "main"?

Reply:
V template máme master, default github vytvára main. Nechajme main a zmeňme to v template. Treba ísť s dobou, byť poloticky korektný a zbaviť sa pozostatkov otroctva

Reply:
OK, vrátil jsem to zpět na 'main' :-)

Reply:
Je to zmenené na viacerých miestach